tests/test_downloadermiddleware_httpcompression.py

```diff
@@ -26,10 +26,23 @@
 
 class HttpCompressionTest(TestCase):
 
+    def create_spider_mw(self
+                         ,compression_enabled=True
```

Comment:
:lipstick: Having the commas at the beginning feels off to me :slightly_frowning_face:

Reply:
Gotcha, wasn't sure what was preffered and I like to have operators at the beginning of a new line instead of trailing on the previous lines so I carried that over to the comas.